splunk_add_on_ucc_framework/modular_alert_builder/build_core/cim_actions.py

```diff
@@ -127,7 +126,7 @@ def message(self, signature, status=None):
         status  = status or self.action_status or ''
         message = ModularAction.DEFAULT_MESSAGE % (signature or '', self.action_name or '', self.search_name or '', self.sid or '', self.orig_sid or '', self.rid or '', self.orig_rid or '', self.app or '', self.user or '', self.action_mode or '', status)
         ## prune empty string key-value pairs
-        for match in re.finditer('[A-Za-z_]+=\"\"(\s|$)', message):
+        for match in re.finditer('[A-Za-z_]+=\"\"(\\s|$)', message):
```

Comment:
This change in the regex will match literal `\` as it's escaped. Is this updated by `pyupgrade`?